### PR TITLE
Update recipes to NXP kirkstone-5.15.32-2.0.0

### PR DESCRIPTION
**Pull request by chrisdimich**

### CODE REVIEW

File: recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
+From 1ba241438dcc14852017757451521f5e27690290 Mon Sep 17 00:00:00 2001
+From: Cristinel Panfir <cristinel.panfir@nxp.com>
+Date: Mon, 2 May 2022 15:18:46 +0000
+Subject: [PATCH] Findtim-vx.cmake: Fix LIBDIR for multilib environment
+
```
Comment (otavio) [inline recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch:5]: Upstream-Status field, please.

File: recipes-libraries/tensorflow-lite/tensorflow-lite-host-tools_2.8.0.bb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+CXXFLAGS += "-fPIC"
+
+do_configure[network] = "1"
+do_configure:prepend() {
+    export HTTP_PROXY=${http_proxy}
+    export HTTPS_PROXY=${https_proxy}
+    export http_proxy=${http_proxy}
+    export https_proxy=${https_proxy}
+}
```
Comment (otavio) [inline recipes-libraries/tensorflow-lite/tensorflow-lite-host-tools_2.8.0.bb:29]: Please fix the missed newline.

File: recipes-libraries/tensorflow-lite/tensorflow-lite-vx-delegate_2.8.0.bb [hunk 3]
```diff
@@ -32,6 +33,7 @@ EXTRA_OECMAKE += " \
 
 CXXFLAGS += "-fPIC"
 
+do_configure[network] = "1"
```
Comment (otavio) [inline recipes-libraries/tensorflow-lite/tensorflow-lite-vx-delegate_2.8.0.bb:36]: Is this required? if so, please add a comment explaining.

**Review by otavio — CHANGES_REQUESTED** [review]

File: recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch (hunk 1 above)
Comment (chrisdimich) [reply recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch:5]: What do you mean by this? I pulled patch directly from NXP BSP:

https://source.codeaurora.org/external/imx/meta-imx/tree/meta-ml/recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch?h=kirkstone-5.15.32-2.0.0

Comment (otavio) [reply recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch:5]: great. But we need to have the header. See:

https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations:_Upstream-Status

File: recipes-libraries/deepview-rt/deepview-rt_2.4.42-aarch64.bb [hunk 4]
```diff
@@ -0,0 +1,69 @@
+DESCRIPTION = "This package includes the updated and experimental ModelRunner for TensorFlow Lite and ARM NN. Also in this repository is a pre-release of DeepViewRT with support for the OpenVX backend."
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d3c315c6eaa43e07d8c130dc3a04a011"
```
Comment (chrisdimich) [inline recipes-libraries/deepview-rt/deepview-rt_2.4.42-aarch64.bb:3]: @otavio actually this md5 is not valid for latest EULA (not in fsl-eula-unpack.bbclass). Any plans to add 5.15.5-2.0.0 license?

Comment (otavio) [reply recipes-libraries/deepview-rt/deepview-rt_2.4.42-aarch64.bb:3]: https://github.com/Freescale/meta-freescale/pull/1149

**Comment by chrisdimich** [comment]
@otavio Fixed recipes as per your comments and also added a few more commits. For deepview-rt to build with the BSP, we need https://github.com/Freescale/meta-freescale/pull/1149 pushed through as you know.

**Review by otavio — APPROVED** [review]

**Review by otavio — APPROVED** [review]

**Pull request merged**